zebra-consensus/Cargo.toml

```diff
@@ -30,11 +33,10 @@ tower-batch = { path = "../tower-batch/" }
 zebra-chain = { path = "../zebra-chain" }
 zebra-state = { path = "../zebra-state" }
 zebra-script = { path = "../zebra-script" }
-pairing = "0.18.0"
+wagyu-zcash-parameters = "0.2.0"
```

Comment:
Do we need to pull in all 7 wagyu sub-crates? They're pretty large.

Does the redundant data get optimised away in release builds?